tests/network/vmi_istio.go

```diff
@@ -412,6 +412,11 @@ var _ = SIGDescribe("[Serial] Istio", func() {
 					registryOnlySidecar := generateRegistryOnlySidecar()
 					_, err = virtClient.DynamicClient().Resource(sidecarRes).Namespace(util.NamespaceTestDefault).Create(context.TODO(), registryOnlySidecar, metav1.CreateOptions{})
 					Expect(err).ToNot(HaveOccurred())
+
+					serviceEntryRes := schema.GroupVersionResource{Group: networkingIstioIO, Version: istioApiVersion, Resource: "serviceentries"}
```

Comment:
I expected `serviceEntryRes` to have a result, not a resource. Probably I'm the one to blame, but I find the name you chose a bit confusing.

Also, could we make the `"serviceentries"` a const ?

Reply:
Renamed the serviceEntryRes to serviceEntryGVK, hope it's more clear.
Added consts for the kind plurals